services/crates/crates-dependents.tester.js

```diff
@@ -0,0 +1,16 @@
+import { isMetric } from '../test-validators.js'
+import { createServiceTester } from '../tester.js'
+export const t = await createServiceTester()
+
+t.create('dependent count').timeout(10000).get('/tokio.json').expectBadge({
+  label: 'dependents',
+  message: isMetric,
+})
+
+t.create('dependent count (nonexistent package)')
+  .timeout(10000)
```

Comment:
Do we expect this to be really slow in the not found case? I tried a couple of calls and they seemed to return a not found response pretty quickly

Reply:
I've taken the test code from `librariesio-dependents.service.js` and left the timeout pretty much as is. I don't think it should take that long to respond. Maybe a second is good enough? What do you think?

Reply:
The default timeout is 4 seconds. In some cases we need to raise it if the upstream endpoint is really slow.

Lets remove this timeout statement and go with the default.

Reply:
It's done. :+1: